### ISSUE TITLE
Feature request: Status of how many layers it's loading?

### ISSUE THREAD

**Issue by cdrage** [containers/podman-desktop-extension-layers-explorer#33]
I have a super-large (10GB) private container image that I use with bootc:

![Screenshot 2024-08-21 at 10 58 47 AM](https://github.com/user-attachments/assets/cb8c1459-ef0d-4c4b-835a-49909f7c1847)

I find that the loading takes a long time, a good 70 seconds.

Any way to calculate how many layers there are on initial loading and have a progress bar of say.. loading 1 of 20 layers or something?

**Comment by feloy** [comment]
Yes, I would like to add such information. For this, the API will need to be extended to be able to have some feedback information from the extension.
